test/Dispatcher.proof.t.sol

```diff
@@ -48,6 +62,14 @@ contract DispatcherIbcWithRealProofs is IbcEventsEmitter, ProofBase {
         dispatcher.openIbcChannel(mars, ch1, ChannelOrder.NONE, false, connectionHops1, ch0, proof);
     }
 
+    function test_ibc_channel_open_dapp_revert() public {
+        Ics23Proof memory proof = load_proof("/test/payload/channel_try_pending_proof.hex");
+
+        vm.expectEmit(true, true, true, true);
+        emit OpenIbcChannelError(address(revertingStringMars));
+        dispatcher.openIbcChannel(revertingStringMars, ch1, ChannelOrder.NONE, false, connectionHops1, ch0, proof);
+    }
+
```

Comment:
these tests belong to `Dispatcher.t.sol` since they don't need to use proofs at all.

Reply:
same with the other ones in the suite.

Reply:
ah yes, should be fixed now!